### PR TITLE
Upgrade flutter version to `v3.7.5`

### PR DESCRIPTION
**Pull request by dab246**
- [x] Add customize padding for child on keyboard
- [x] Upgrade flutter version to `v3.7.5`
- [x] Upgrade latest version for dependencies

### CODE REVIEW

File: example/ios/Runner/Info.plist [hunk 1]
```diff
@@ -45,5 +45,7 @@
 	<false/>
 	<key>CADisableMinimumFrameDurationOnPhone</key>
 	<true/>
+	<key>UIApplicationSupportsIndirectInputEvents</key>
+	<true/>
```
Comment (hoangdat) [inline example/ios/Runner/Info.plist:49]: what is it?

File: pubspec.yaml [hunk 2]
```diff
@@ -1,26 +1,26 @@
 name: rich_text_composer
 description: A new Flutter project.
```
Comment (hoangdat) [inline pubspec.yaml:2]: please say sth about this project :))

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -28,9 +28,6 @@ jobs:
       # Install Flutter
       - name: Install Flutter
         uses: subosito/flutter-action@v2
-        with:
-          flutter-version: "3.0.5"
-          channel: "stable"
```
Comment (hoangdat) [inline .github/workflows/ci.yml:33]: @tk-nguyen we need to have a specific version, am I right?

Comment (tk-nguyen) [reply .github/workflows/ci.yml:33]: Yep.

File: example/ios/Runner/Info.plist (hunk 1 above)
Comment (dab246) [reply example/ios/Runner/Info.plist:49]: From `iOS 13.4+` Info.plist automatically adds this. Please refer at [here](https://developer.apple.com/documentation/bundleresources/information_property_list/uiapplicationsupportsindirectinputevents)

File: .github/workflows/ci.yml (hunk 3 above)
Comment (dab246) [reply .github/workflows/ci.yml:33]: IMO, If we don't specify it, it will use the latest stable version

File: pubspec.yaml (hunk 2 above)
Comment (dab246) [reply pubspec.yaml:2]: Please help me desc for it @hoangdat

File: .github/workflows/ci.yml (hunk 3 above)
Comment (tk-nguyen) [reply .github/workflows/ci.yml:33]: If that's what you want, it's fine. It will use the latest version every time it runs, so keep that in mind.

**Review by hoangdat — APPROVED** [review]

**Pull request merged**